soh/soh/OTRGlobals.cpp

```diff
@@ -1479,27 +1495,24 @@ extern "C" int CopyScrubMessage(u16 scrubTextId, char* buffer, const int maxBuff
     return CopyStringToCharBuffer(scrubText, buffer, maxBufferSize);
 }
 
-extern "C" int Randomizer_CopyAltarMessage(char* buffer, const int maxBufferSize) {
-    const std::string& altarText = (LINK_IS_ADULT) ? OTRGlobals::Instance->gRandomizer->GetAdultAltarText()
-                                                   : OTRGlobals::Instance->gRandomizer->GetChildAltarText();
-    return CopyStringToCharBuffer(altarText, buffer, maxBufferSize);
+extern "C" CustomMessageEntry Randomizer_CopyAltarMessage() {
+    return (LINK_IS_ADULT) ? CustomMessageManager::Instance->RetrieveMessage(Randomizer::hintMessageTableID, 0x7088)
+                           : CustomMessageManager::Instance->RetrieveMessage(Randomizer::hintMessageTableID, 0x7040);
```

Comment:
the hex values here feel a little like magic numbers, any chance we could use an enum or something for the known ones?

Reply:
Possibly. I think those particular values match the vanilla `textID`s that they are overriding, but some clarity could be good. Might be a `#define` rather than an enum though since we're calling out select values and with enums the positioning of any later added values would be important.

Reply:
with an enum you can do
```cpp
enum blarg {
    a = 3,
    b = 20,
    c = 9001
}
```
so positioning doesn't need to matter

Reply:
Well they do have to go in order with enums right? Although that's not exactly difficult to do in code if you're adding an explicit textId anyway. You're right, enum seems cleaner.

Reply:
Fixed in 5a97e9f